.travis.yml

```diff
@@ -1,3 +1,7 @@
 language: java
+jdk:
+  - oraclejdk8  
 
-jdk: oraclejdk8
+# Please enable it. Normally CI should deploy on the snapshot repo, not just
```

Comment:
I created a follow-up ticket for me: https://github.com/harti2006/neo4j-server-maven-plugin/issues/3.
Please remove the comments.

Reply:
Going to remove them and then I'll let you to change your main repo according to that ticket. On our side, I'm happy to re-target our dependencies to your repo, once recent changes have reached it.